src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -160,6 +160,7 @@ const SampleChatInfoScreen = () => {
   const {channelInfo, goBack, onContactPressed} = useAnonymousChatInfoScreenHook();
 
   const [isLoadingMembers] = React.useState<boolean>(false);
+  // TODO: Change this into useUserAuthHook
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The comment suggests replacing `useProfileHook` with `useUserAuthHook`. If `useUserAuthHook` provides the same functionality and more, it would be beneficial to replace it for consistency across the application. However, ensure that `useUserAuthHook` does provide all necessary data and functionality before making this change.
```diff
-   const {signedProfileId} = useProfileHook();
+   const {signedProfileId} = useUserAuthHook();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->